projects/ngx-mask-lib/src/lib/ngx-mask-applier.service.ts

```diff
@@ -539,7 +547,7 @@ export class NgxMaskApplierService {
         }
         let res = `${this.prefix}${onlySpecial ? '' : result}${this.suffix}`;
         if (result.length === 0) {
-            res = `${this.prefix}${result}`;
+            !this.dropSpecialCharacters ? (res = `${this.prefix}${result}`) : (res = `${result}`);
```

Comment:
could this be 
`res = !this.dropSpecialCharacters ?`${this.prefix}${result}` : `${result}`;`